src/updatedb.py

```diff
@@ -150,11 +150,12 @@ def uploadResults(data, branch, revision, date):
                    
         testtype = r[i("ref_data_name")].split()[-1]
         if r[i("build_system_type")] == "taskcluster":
-            #TODO: this is fragile, current platforms as of Jan 26, 2016 we see in taskcluster
             pmap = {"linux64": "Linux64",
                     "linux32": "Linux32",
                     "osx-10-7": "MacOSX64",
                     "gecko-decision": "gecko-decision",
+                    "b2g-device-image": "B2G",
+                    "mulet-linux64": "Mulet",
```

Comment:
I would like to intentionally skip these platforms, so if we add them, we should consider doing that so we can ignore them properly.  All B2G and Mulet builds are either being shut down or are only running until a certain time on a calendar is hit and then they will be shutdown.  Can you outline what coverage we plan to get with this?